### ISSUE TITLE
Pressing enter in completion popup inserts new line

### ISSUE THREAD

**Issue by bgr** [VsVim/VsVim#2788]
**Describe the bug**
After using Ctrl-P / Ctrl-N and choosing a completion by pressing Enter, additional new line gets inserted.

**To Reproduce**
I don't know how to reproduce what initially caused this to start happening, VsVim was behaving correctly on my machine before (more info in "Additional context" below). Now that it's in this state, I can reproduce it every time I use Ctrl-P/Ctrl-N and press Enter.

**Expected behavior**
No additional newlines get inserted.

**Screenshots**
![Imgur image](https://i.imgur.com/MnOB7dW.gif)

**Environment (please complete the following information):**

- Visual Studio version: Community 2019 - 16.4.3
- VsVim version: 2.8.0.0
- Programming Language: C# (Unity)
- Check(Type 'x') any that are installed:
    - [ ] ReSharper
    - [ ] Visual Assist

**Additional context**

* I tried disabling & enabling (with restarting Visual Studio) VsVim, didn't help.
* I disabled all other extensions I had installed, didn't help.
* At one point I logged into my Microsoft account from VS on another machine, and that messed up keyboard shortcuts on my main work machine where I'm experiencing this problem. This might be the point where this started happening, but I'm not 100% sure.
* I postponed writing this but at this point I've spent too much time on it and have to ask for help since it's driving me crazy, I know it might not be a VsVim bug (seems like VS handles the enter keypress whereas it should be ignored), but I tried fixing it myself and read many open & closed issues and didn't manage to find anything.

**Comment by sqlkoala** [comment]
> **To Reproduce**
> I don't know how to reproduce what initially caused this to start happening, VsVim was behaving correctly on my machine before (more info in "Additional context" below).

Hi,
what do you mean by "correctly"? Did it change after an update of Vsvim or VS (that would be my guess)?

About the "problem" itself: this is the normal vim behavior that vsvim emulates. But you have a point. VS completion and vsvim's ctrl-n/ctrl-p are different at this point:
- VS consumes the key used to pick a completion (tab, enter)
- vim does not consume it; it puts them in den document

For users who are accustomed heavily to the VS way, there could be a configuration option.

**Comment by bgr** [comment]
You're right, I've compared it now to VsVim on another computer (that one has Visual Studio Community 2017 installed, VsVim updated to 2.8.0.0), and found what actually triggered my muscle memory confusion.

That VsVim (as well as regular vim) completes the word while cycling through the list. The one on my development computer doesn't, this is how it looks like:

![Imgur gif](https://i.imgur.com/IWnSBVg.gif)

I've compared the VsVim settings (from Visual Studio's "Tools > Options" menu) on the two computers and they seem identical.

I can open a new issue for this if needed.
